packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -1005,13 +1020,13 @@ class _DialState extends State<_Dial> with SingleTickerProviderStateMixin {
     switch (widget.mode) {
       case _TimePickerMode.hour:
         if (widget.is24h) {
```

Comment:
how does is24h interact with the new flag?

Reply:
It seems weird to be able to have the double ring with the 12h mode or the single ring with 24h mode...